cli/import-jdl.js

```diff
@@ -229,14 +229,15 @@ const generateEntityFiles = (generator, entity, inFolder, env, shouldTriggerInst
         });
     } else {
         /* Traditional entity only generation */
+        const callback = isLastEntityToGenerate ? done : () => {};
         env.run(
             command,
             {
                 ...options,
                 force: options.force || !options.interactive,
                 'skip-install': !shouldTriggerInstall
             },
-            done
+            callback
```

Comment:
This is minor but the maybe we can simplify this by just removing the constant and callback and adding; 

```js
if (isLastEntityToGenerate) 
    done
```

Reply:
@SudharakaP do you mean ? 
`done`
=>
`isLastEntityToGenerate ? done : () => {}`

Reply:
because we can't write your code like this

Reply:
@qmonmert : Got it; you are correct. I didn't see it's an argument. :smile: